src/interactive/results.ts

```diff
@@ -195,10 +195,27 @@ export function activate(context) {
             gotoNextFrame(frameArg.frame)
         }),
         vscode.commands.registerCommand('language-julia.gotoLastFrame', gotoLastFrame),
-        vscode.commands.registerCommand('language-julia.clearStackTrace', clearStackTrace)
+        vscode.commands.registerCommand('language-julia.clearStackTrace', clearStackTrace),
+        vscode.window.onDidChangeTextEditorSelection(changeEvent => updateResultContextKey(changeEvent))
     )
 }
 
+function updateResultContextKey(changeEvent: vscode.TextEditorSelectionChangeEvent) {
+    if (changeEvent.textEditor.document.languageId !== 'julia') {
+        return
+    }
+    for (const selection of changeEvent.selections) {
+        for (const r of results) {
+            if (isResultInLineRange(changeEvent.textEditor, r, selection)) {
+                vscode.commands.executeCommand('setContext', 'juliaHasInlineResult', true)
+                return
+            }
+        }
+    }
+    vscode.commands.executeCommand('setContext', 'juliaHasInlineResult', false)
```

Comment:
A bit unfortunate that `setContext` isn't documented: https://github.com/microsoft/vscode/issues/46445

Reply:
I actually used this in my documentation pane PR , and it would be appreciated if you move this into utils.ts or somewhere as I did ?
https://github.com/julia-vscode/julia-vscode/blob/939065d64e9127166321df09aab6cf32783f08ac/src/utils.ts#L7-L9